scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java

```diff
@@ -3192,6 +3189,7 @@ public List<JobLabelInfo> newLabels(List<String> labels) throws LabelConflictExc
             JobLabel jobLabel = executeReadWriteTransaction(session -> {
                 JobLabel labelData = JobLabel.createJobLabel(label);
                 session.save(labelData);
+                logger.info("Label " + label + " has been created");
```

Comment:
it is better to put this log message inside the frontend caller method.
This way, you can include in the log message the user which created the label

Reply:
For some operation we don't have the label name in the frontend caller method. We receive the Id as request param

Reply:
alright, in that case, you could add the intiator as a parameter to the database methods. This way you would have both information when printing the logs.